### DISCUSSION TITLE
Ctrl S saves as HTML

### DISCUSSION THREAD

**Discussion by deuts** [docmost/docmost#449] [Help / Questions] [answered]
It has always been a habit of mine to press `Ctrl S` every time I'm typing in office documents and text editors. In Docmost, however, it prompts me to save the page as html. I know this is a petty issue but can we suppress that prompt instead?

**Comment by Philipinho** [comment, marked as answer]
Fixed https://github.com/docmost/docmost/pull/450.
It will be available in the next release.

**Comment by neuhxy90** [comment]
In macOS, `Command + S` still triggers saving as HTML.
